ci/lava/tests/test-core-component.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+Script to install and run core component tests on the DUT.
+
+This pytest script consists of a test_setup function that installs the
+necessary pre-built images and builds and installs images for firmware
+testing onto the DUT. It also copies the entire mbl-core clone onto the DUT.
+
+The test_component function runs pytest on the DUT via the mbl-cli on the
+mbl-core copy.
+
+"""
+import pytest
+import re
+import subprocess
+
+
+class Test_Core_Component_DUT:
+    """Class to encapsulate the testing of core components on a DUT."""
+
+    def test_setup(self, dut_addr, execute_helper):
+        """Copy the test specific parts, generating items as required."""
+        execute_helper._send_mbl_cli_command(
+            [
+                "put",
+                "/tmp/user-sample-app-package_1.0_any.ipk",
+                "/scratch/app-lifecycle-manager-test-package_1.0_any.ipk",
```

Comment:
Again - this should either have class strings (or globals) that define /tmp and /scratch - or have defaults and paramters to control them.

Reply:
Changed the hard coded paths to be parameters that can be specified to pytest via the job pipelines.